tests/api/test_security.py

```diff
@@ -13,6 +13,7 @@ def test_main_page(self, session, flask_api_url):
         assert response.status_code == 200
 
     def test_login_fake_user(self,session, flask_api_url):
+        session = requests.Session()
```

Comment:
Yes, but you shouldn't use the conftest fixture "session" as a method argument. You create your own session, there is no need to get that of the fixture.